src/main.py

```diff
@@ -54,6 +54,13 @@ def add_config_file(grp, is_required):
         action='store_true',
         help='Start as long-running client node. Waits for master '
              'to send experiment configuration, and runs them.')
+    group_train.add_argument(
+        '--checkpoint-folder',
+        '-c',
+        type=str,
+        required=False,
+        dest='checkpoint_folder',
+        help='Folder of the experiment to recover from its checkpoints.')
```

Comment:
Unclear english, is this the folder for restarting or saving checkpoints. I like to make an example in the help of how to use the flage.g `E.g. ./checkpoints`